h/models/token.py

```diff
@@ -48,10 +58,13 @@ class Token(Base, mixins.Timestamps):
     #: A NULL value means it is a developer token.
     authclient = sqlalchemy.orm.relationship('AuthClient')
 
-    def __init__(self, **kwargs):
-        super(Token, self).__init__(**kwargs)
+    def __init__(self, expires=None, **kwargs):
+        super(Token, self).__init__(expires=expires, **kwargs)
         self.regenerate()
 
+        if expires:
+            self.refresh_token = self.refresh_token_prefix + _token()
```

Comment:
I don't think the refresh token needs a prefix. The main purpose of the token prefix is for a human to be able to quickly and easily identify that the string they're looking at is an API token. The refresh token should never be used by a human -- it's a string that only machines have to worry about.

In addition, I'd suggest that for ease of identifying that this is doing the correct thing, we use `h.security.token_urlsafe` to generate the refresh tokens. (Perhaps we want to switch to using this for API tokens at some point in the future, too.)